app/src/main/java/com/snatik/matches/fragments/ThemeSelectFragment.java

```diff
@@ -12,12 +10,14 @@
 import android.view.animation.DecelerateInterpolator;
 import android.widget.ImageView;
 
-import com.snatik.matches.R;
-import com.snatik.matches.common.Memory;
-import com.snatik.matches.common.Shared;
-import com.snatik.matches.events.ui.ThemeSelectedEvent;
-import com.snatik.matches.themes.Theme;
-import com.snatik.matches.themes.Themes;
+import com.obosapps.matches.R;
```

Comment:
please change all imports back to `com.snatik.matches`. Otherwise it won't compile.